WTWSMS/history/titles/c_belgrade.txt

```diff
@@ -7,4 +7,7 @@
 }
 540.1.1={
 	holder=6831 
+}
+574.1.1={
+    holder=1310016
```

Comment:
Singidunum is the city which became Belgrade, not Sirmium.

Reply:
I know but it says that he was Duke of Sirmium (Which I decided to replace with the duchy of Moesia Superior, as IMO it is more optimal to give) and Belgrade is under him.

Reply:
So count rather?

Reply:
Maybe, but I gave him duchy title to share the land a bit more.